actions/src/test/java/com/redhat/service/bridge/actions/webhook/WebhookInvokerTest.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.service.bridge.actions.webhook;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import org.junit.jupiter.api.Test;
+
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class WebhookInvokerTest {
+
+    public static final String TEST_EVENT = "{\"specversion\":\"1.0\",\"type\":\"TestType\",\"source\":\"/test/src\",\"id\":\"1234\"}";
+
+    @Test
+    void test() throws IOException, InterruptedException {
+        MockWebServer mockWebServer = new MockWebServer();
```

Comment:
@kostola Would be good to keep things standardised and rely on WireMock here for capturing and inspecting HTTP requests. It's something we're already using in other tests. Take a look at `AbstractShardWireMockTest` for example where we use/configure `WireMockServer`

Reply:
Ok thanks for the hint. I will take a look at how we already use WireMock.

Reply:
Switched to WireMock :+1: